packages/tests/src/create.ts

```diff
@@ -2,32 +2,46 @@ import { LAMPORTS_PER_SOL } from "@solana/web3.js";
 import chai from "chai";
 import chaiAsPromised from "chai-as-promised";
 import { createCryptid } from "./util/cryptid";
-import { initializeDIDAccount } from "./util/did";
-import { fund, createTestContext } from "./util/anchorUtils";
+import { getDidAccount, initializeDIDAccount } from "./util/did";
+import { fund, createTestContext, Wallet } from "./util/anchorUtils";
 
 chai.use(chaiAsPromised);
 const { expect } = chai;
 
-describe("create", () => {
-  const { authority, provider } = createTestContext();
+const testCases = [
+  {
+    name: "generative",
```

Comment:
we should probably emphasise "generative DID" here, since the test cases themselves revolve around generative/non-generative cryptids